IsUserInActiveSubsetActivity/index.ts

```diff
@@ -0,0 +1,44 @@
+﻿import { createTableService } from "azure-storage";
+
+import { getIsUserATestUser } from "../utils/checkTestUsers";
+import { getConfigOrThrow } from "../utils/config";
+import { createActivity } from "../utils/durable/activities";
+import { getIsInActiveSubset } from "../utils/featureFlags";
+
+import {
+  ActivityBodyImpl,
+  ActivityInput,
+  activityResultSuccessWithValue,
+  ActivityResultSuccessWithValue,
+  getActivityBody
+} from "./handler";
+
+const config = getConfigOrThrow();
+
+const tableService = createTableService(
+  config.BETA_USERS_STORAGE_CONNECTION_STRING
+);
+
+export {
+  ActivityBodyImpl,
+  ActivityInput,
+  ActivityResultSuccessWithValue,
+  activityResultSuccessWithValue
+};
+
+export const activityName = "IsUserInActiveSubsetActivity";
+
+const activityFunction = getActivityBody(
+  getIsInActiveSubset(
+    getIsUserATestUser(tableService, config.BETA_USERS_TABLE_NAME)

```

Comment:
Have you considered using an input binding to read from the table? 
https://docs.microsoft.com/it-it/azure/azure-functions/functions-bindings-storage-table-input?tabs=javascript

It will make code way more simple (also, considering this piece will be dismissed after rollout, the less we write, the easier we can dismiss it)

Reply:
No, actually. I agree, much better this way